src/main/java/org/usfirst/frc/team5687/robot/commands/PrimeBoulder.java

```diff
@@ -0,0 +1,40 @@
+package org.usfirst.frc.team5687.robot.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import org.usfirst.frc.team5687.robot.Constants;
+
+import static org.usfirst.frc.team5687.robot.Robot.intake;
+
+/**
+ * Command to prime the intake or move the boulder back to primed position
+ * @author wil
+ */
+public class PrimeBoulder extends Command {
+
+    public PrimeBoulder() {
+        requires(intake);
+    }
+
+    @Override
+    protected void initialize() {
+    }
+
+    @Override
+    protected void execute() {
+        intake.setSpeed(Constants.Intake.PRIME_SPEED);
+    }
+
+    @Override
+    protected boolean isFinished() {
+        return intake.isPrimeAble();
```

Comment:
We might want to change the method/ constant named to "primed" vs "primable" for clarity.